ci_framework/roles/operator_build/tasks/build.yml

```diff
@@ -14,17 +14,31 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: "{{ operator.name }} - Set default api path"
-  ansible.builtin.set_fact:
-    operator_api_path: "github.com/{{ cifmw_operator_build_org }}/{{ operator.name }}/api"
+- name: "{{ operator.name }} - Get base module name and set the api path"
+  when:
+    - cifmw_operator_build_meta_build
+    - operator.name != cifmw_operator_build_meta_name
+  block:
+    - name: {{ operator.name }} - Read go.mod file contents of Openstack Operator
```

Comment:
does this task name need the quotes because of {{ 

I have had this fail in the past ... but perhaps it is dependent on ansible version or something (i.e. have you run this like that and it is OK?)

Reply:
thanks, marios, fixed it.

I couldn't test earlier as the image build had issues due to missing nova hash(this issue got fixed today)